reindexer/complete_reindex/src/complete_reindex.py

```diff
@@ -2,31 +2,80 @@
 
 import os
 
+from boto3.dynamodb.conditions import Attr
 import boto3
-from botocore.exceptions import ClientError
+
 from wellcome_aws_utils import sns_utils
 
 
-def main(event, _ctxt=None, dynamodb_client=None):
-    print(f'event = {event!r}')
-    dynamodb_client = dynamodb_client or boto3.client('dynamodb')
+def _get_item_version(dynamo_item):
+    dynamo_item_version = 1
+
+    if 'version' in dynamo_item:
+        dynamo_item_version = dynamo_item['version']
+
+    return dynamo_item_version
```

Comment:
Easier would be to use the `get` method on Python dicts: `dict.get(key[, default]`

i.e.

```python
dynamo_item.get('version', 1)
```